src/alpha-lab/phone-neon.md

```diff
@@ -10,42 +10,33 @@ tags: [Neon, Cloud]
 
-### How we used them
-We first used the Reference Image Mapper enrichment to create a 3D model of a phone positioned on a desk and subsequently to map gaze onto a 2D image of the phone-on-desk. Then we used Alpha Lab tutorials to process the exported results, generate advanced visualisations, and compute outcome metrics.
+Subsequently, we exported the data to analyse offline for in-depth exploration. For the offline analysis, we used existing Alpha Lab tutorials that allowed us to [generate scanpaths](/alpha-lab/scanpath-rim/), [define areas of interest (AOIs) and calculate gaze metrics](/alpha-lab/gaze-metrics-in-aois/), and [map gaze onto dynamic content](/alpha-lab/map-your-gaze-to-a-2d-screen/). 
 
-:::tip
-:bulb: 
-When preparing the Reference Image Mapper Enrichment, make sure your phone is stable on a phone mount or stand. The 
-scanning video needed for this tool requires relatively static features in the environment. If there is a lot of movement 
-or the objects change in appearance or shape, the mapping can fail. More on this [in the docs](/enrichments/reference-image-mapper#setup/)! 
-:::
+The following section presents the results of our analyses, each accompanied by a brief description for clarity.
 
-## Gaze behaviour on mobile apps: Insights from Neon
 ### Heatmaps and scanpaths
-What catches your attention and how do you visually navigate through the interface/page of a mobile app?
+Two valuable visualisations for understanding gaze behaviour on mobile screens are heatmaps and scanpaths (illustrated in the left and right panels below). 
+
+Heatmaps visually represent the app areas that receive the most attention, with warmer colours denoting higher fixation rates. These heatmaps can be easily generated within Pupil Cloud using the Reference Image Mapper.
 
-Two visualisations that help to illustrate these patterns are heatmaps and scanpaths (left and right panel below). Heatmaps show the areas of the app that receive the most attention, with warmer colours indicating more fixations, and can be generated natively in Pupil Cloud. Meanwhile, scanpaths trace the path of the eye movements, showing the sequence of fixations and eye movements that occur during the visual exploration. The circle size of the scanpath below reflects fixation duration: The bigger the circle, the longer the user fixated on this area of the screen. Use the scanpath tutorial linked above to generate this visualisation.
+Meanwhile, scanpaths depict the sequence of eye movements and fixations during visual exploration. The circle size in the scanpath corresponds to fixation duration; larger circles indicate longer user focus on specific screen areas. For those interested in generating these visualisations, our [scanpath tutorial](/alpha-lab/scanpath-rim/) provides a step-by-step guide.
 
 <div class="mcontainer">
```

Comment:
Let's move these images such that they appear first, underneath the section header

Reply:
Follow-up question. I can't remember if we already discussed this. But the heatmap seems not to correspond to the scanpath visualisation. That is, there are no fixations shown on the hottest area of the heatmap. Is that not counterintuitive?